diff --git a/src/main/java/FirstFile.java b/src/main/java/FirstFile.java
index 9a78dc2..f0b5ac6 100644
--- a/src/main/java/FirstFile.java
+++ b/src/main/java/FirstFile.java
@@ -6,8 +6,8 @@
 	
 	private HashMap map;
 	
-	public FirstFile() {
-		this.map = new HashMap<>();
+	public FirstFile(HashMap inputMap) {
+		this.map = inputMap;
 	}
 	
 	public void method11(String input1) {
diff --git a/src/main/java/SecondFile.java b/src/main/java/SecondFile.java
index 06ca781..c6654ee 100644
--- a/src/main/java/SecondFile.java
+++ b/src/main/java/SecondFile.java
@@ -6,8 +6,8 @@
 	
 	private HashMap map;
 	
-	public SecondFile() {
-		this.map = new HashMap<>();
+	public SecondFile(HashMap inputMap) {
+		this.map = inputMap;
 	}
 	
 	public String method21() {
