diff --git a/src/main/java/FirstFile.java b/src/main/java/FirstFile.java
index 578cbf8..9a78dc2 100644
--- a/src/main/java/FirstFile.java
+++ b/src/main/java/FirstFile.java
@@ -6,6 +6,10 @@
 	
 	private HashMap map;
 	
+	public FirstFile() {
+		this.map = new HashMap<>();
+	}
+	
 	public void method11(String input1) {
 		System.out.println(input1);
 	}
diff --git a/src/main/java/SecondFile.java b/src/main/java/SecondFile.java
index ba96a2f..06ca781 100644
--- a/src/main/java/SecondFile.java
+++ b/src/main/java/SecondFile.java
@@ -6,6 +6,10 @@
 	
 	private HashMap map;
 	
+	public SecondFile() {
+		this.map = new HashMap<>();
+	}
+	
 	public String method21() {
 		return "Hello World!";
 	}
